lints.rs

```diff
@@ -1,4 +1,4 @@
-// BEGIN - Embark standard lints v5 for Rust 1.55+
+// BEGIN - Embark standard lints v6 for Rust 1.55+
```

Comment:
Were they default-warn on v1.55 as well?

Reply:
no I don't think so, but I didn't want to say that this lint set technically requires 1.59 yet as it doesn't. and it is a bit of a stopgap release until we do v7 that does require 1.59 with new lints.